addons/calendar/models/calendar_event.py

```diff
@@ -556,6 +563,29 @@ def action_sendmail(self):
                 meeting.attendee_ids._send_mail_to_attendees('calendar.calendar_template_meeting_invitation')
         return True
 
+    def action_mass_mailing(self):
+        partners_ids = self.mapped('partner_ids')
+        if not partners_ids:
+            raise UserError(_("There are no attendees on these events"))
+        compose_form = self.env.ref('mail.email_compose_message_wizard_form', False)
+        default_partners = partners_ids and partners_ids.ids
+        compose_ctx = dict(
```

Comment:
You should not open a composer without model and ids ... this will lead to a lot of issues as we always rely on a model to call some methods. Mail composer is not meant to "send an email to some partners". It is meant to be used on top of business documents.